LinkBase/LinkBase/HudView.swift

```diff
@@ -7,3 +7,64 @@
 //
 
 import Foundation
+import UIKit
+
+class HudView: UIView{
+    var text = ""
+    
+    class func hud(inView view: UIView, animated: Bool) -> HudView {
+        
+        let hudView = HudView(frame: view.bounds)
+        hudView.isOpaque = false
+        
+        view.addSubview(hudView)
+        view.isUserInteractionEnabled = true
+        
+        hudView.show(animated: animated)
+        return hudView
+    }
+    
+    override func draw(_ rect: CGRect) {
+        
+        let boxWidth: CGFloat = 96
```

Comment:
This is not based on auto layout settings, it's just a standard size for the hud view.  We can adjust the size (perhaps also on different view controllers if we choose), but the size of the Rect should be constant and not varying in different sizes